docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3.7"
+
+services:
+  pgdb:
+    build: env/postgres
+    image: pgdb
```

Comment:
This is just like tagging a container with a name for easy access, I just chose `pgdb` for the postgres container.